src/main/java/entityLayer/CommonSchedule.java

```diff
@@ -0,0 +1,18 @@
+package entityLayer;
+
+public class CommonSchedule implements Schedule {
+    @Override
+    public boolean addAvailability(TimeBlock availableBlock) {
+        return false;
+    }
+
+    @Override
+    public boolean removeAvailability(TimeBlock unavailableBlock) {
+        return false;
+    }
+
+    @Override
+    public Curriculum getCurriculum(int curriculumID) {
+        return null;
+    }
+}
```

Comment:
method for changeAvailability

Reply:
I'll go ahead and put that in, but I'm going to hold off a while on actually implementing. We've got 10 days until the deadline as of me writing this reply, and creating a "changeavailability" method will require also changing every single curriculum's TimeManager objects to fit into the new availability, which will probably also have the user have to review it, etc, etc.While I'd love to implement that functionality, it just isn't as pressing as our more foundational functionalities, and poses way too many complications for us to begin considering implementing it this early into the process.

I'll type it into the interface just to have it, but I'll make a note in the documentation that it has yet to be implemented, and that the implementation will be something to be decided on later.